lib/core/themes/text_styles.dart

```diff
@@ -49,4 +49,45 @@ class TextStyles {
     fontWeight: FontWeight.bold,
     color: AppColors.white,
   );
+  static TextStyle textStyleGradient = TextStyle(
+      fontFamily: "Space Grotesk",
+      fontWeight: FontWeight.w500,
+      fontSize: 12.sp,
+      foreground: Paint()
+        ..shader = LinearGradient(
+          colors: <Color>[
+            AppColors.textGradientColor1,
+            AppColors.textGradientColor2,
+            AppColors.textGradientColor3
+          ],
+        ).createShader(const Rect.fromLTWH(110.0, 70.0, 200.0, 70.0)));
+  static TextStyle textStyle34 = TextStyle(
+    fontSize: 34.sp,
+    fontFamily: "Space Grotesk",
+    fontWeight: FontWeight.w300,
+    color: AppColors.white,);
+
+  static TextStyle textStyle14300 = TextStyle(
+    fontSize: 14.sp,
+    fontWeight: FontWeight.w300,
+    color: AppColors.lightGrey,
+    fontFamily: "Space Grotesk",
+  );
+  static TextStyle textStyleOfWhite16300 = TextStyle(
+    fontSize: 16.sp,
+    fontWeight: FontWeight.w300,
+    color: AppColors.offWhite,
+  );
+  static TextStyle textStyle16400 = TextStyle(
```

Comment:
what is textStyle16400 ?

that's not good name, we need something like ( font16White400w ) or any other descriptive name.